stem/kratos_IO.py

```diff
@@ -1,4 +1,8 @@
+import json
+
 import numpy as np
+from stem.load import Load, PointLoad
+from typing import List, Dict, Any
```

Comment:
order of inputs should be: python imports -> external packages -> project imports, with Enters between the different types

Reply:
i.e.: json, typing -> numpy -> stem

Reply:
Indeed, forgot to run the import optimizer!